.lift.toml

```
@@ -0,0 +1,3 @@
+setup = "scripts/setupLift.sh"
+build = "mvn install -pl !:integrationtests --also-make-dependents"
```

AFAIK Lift is static analysis, so running the tests just adds latency without benefit.

```suggestion
build = "mvn install -pl !:integrationtests --also-make-dependents -DskipTests"
```

---

Yes, I don't mean to merge this PR, this was just to play around with lift application and help Sam's PR.